oem/firstvoices/android/app/src/main/AndroidManifest.xml

```diff
@@ -41,7 +42,8 @@
 
         <activity
             android:name="com.firstvoices.keyboards.MainActivity"
-            android:label="@string/app_name" >
+            android:label="@string/app_name"
+            android:configChanges="orientation|screenSize">
```

Comment:
This is to avoid rotating the device causing MainActivity.onCreate() to re-run and instantiate FVShared() again, causing this assertion to fail

https://github.com/keymanapp/keyman/blob/52d36abb4fd91c289087c9605b60919307b826bd/oem/firstvoices/android/app/src/main/java/com/firstvoices/keyboards/FVShared.java#L103

Reply:
Are there other circumstances in which `onCreate()` can run again? Surely we could fix this by testing if `FVShared()` had already been run in `onCreate()` for added robustness?

Reply:
We should be using a Singleton design pattern :grin:

Reply:
I've updated FVShared to be "more" Singleton. @MakaraSok can you retest Android and see if it's more stable?

@keymanapp-test-bot retest TEST_FV_ANDROID